Saule/Serialization/ResourceSerializer.cs

```diff
@@ -264,12 +267,12 @@ private JObject SerializeAttributes(ResourceGraphNode node)
         {
             var attributeHash = node.Resource.Attributes
                 .Where(a =>
-                    node.SourceObject.IncludesProperty(a.PropertyName))
+                    node.SourceObject.IncludesProperty(a.Name))
```

Comment:
Should this be `ToModelPropertyName(a.Name)`? If people do not use `nameof` when creating their model, would this break?